test/Fhir.Metrics.Tests/Tests/TestMetrics.cs

```diff
@@ -113,14 +114,21 @@ public void MetricWithValidAnnotations()
         [Fact]
         public void MetricWithInvalidAnnotations()
         {
-            try
-            {
-                Metric metric = system.Metric("{{1}}");
-            }
-            catch (Exception e)
-            {
-                Assert.IsType<ArgumentException>(e);
-            }
+            Action act = () => system.Metric("{{1}}");
+            act.Should().Throw<ArgumentException>("Nested annotations are not allowed");
+        }
+
+        [Fact]
+        public void MetricShouldRejectWhitespaces()
+        {
+            Action act = () => system.Metric("/min 1/min {breaths}/min {breath}/min {resp}/min");
+            act.Should().Throw<ArgumentException>("Whitespaces are not allowed in a metric expression");
+
+            act = () => system.Metric(" /min ");
+            act.Should().Throw<ArgumentException>("Whitespaces are not allowed in a metric expression");
```

Comment:
I am wondering whether or not a message about whitespace is understandable to users if the whitespace is just leading or trailing